internal/version/version_test.go

```diff
@@ -0,0 +1,37 @@
+package version
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
To fix the depgaurd error you need to add new packages to this file: https://github.com/ansible/receptor/blob/devel/.golangci.yml#L84